packages/yuudachi/src/functions/cases/transformCase.ts

```diff
@@ -14,6 +14,7 @@ export interface RawCase {
 	target_tag: string;
 	context_message_id: Snowflake | null;
 	ref_id: number | null;
+	report_ref: number | null;
```

Comment:
This should really be called `report_ref_id` in the db and here.

Reply:
Changed in e0717b3